docs/content/en/getting_started/architecture.md

```diff
@@ -20,8 +20,8 @@ dynamic content.
 
 ## Message Broker
 
-The application server sends tasks to a [Message Broker](https://docs.celeryproject.org/en/stable/getting-started/brokers/index.html)
-for asynchronous execution.
+The application server sends tasks to a [Message Broker](https://docs.celeryq.dev/en/stable/getting-started/first-steps-with-celery.html#choosing-a-broker)
+for asynchronous execution. [RabbitMQ](https://www.rabbitmq.com/) is a well established choice.
```

Comment:
We're no longer supporting RabbitMQ and now prefer Redis, as noted [here](https://documentation.defectdojo.com/getting_started/architecture/#message-broker). We can copy that same language here or just leave it out.

```suggestion
for asynchronous execution.
```